src/Plugins/Pagination.js

```diff
@@ -107,27 +107,27 @@ class Pagination {
 		if (hasInclude) {
 			let included = this.data.pagination.include;
 			if (Array.isArray(included)) {
-				return !included.includes(value);
+				return included.includes(value);
 			}
-			return included !== value;
+			return included === value;
 		}
 		if (hasExclude) {
 			let excluded = this.data.pagination.exclude;
 			if (Array.isArray(excluded)) {
-				return excluded.includes(value);
+				return !excluded.includes(value);
 			}
-			return excluded === value;
+			return excluded !== value;
 		}
 
 		// Let's keep this code for backwards compatibility to V2.
 		// TODO remove in 3.0
 		if ("filter" in this.data.pagination) {
 			let filtered = this.data.pagination.filter;
 			if (Array.isArray(filtered)) {
-				return filtered.indexOf(value) > -1;
+				return filtered.indexOf(value) === -1;
 			}
 
-			return filtered === value;
+			return filtered !== value;
 		}
 
 		return false;
```

Comment:
And I think this default return value would be `true` now that the function is inverted?

Reply:
Good catch, will update... 🤦